### PR TITLE
Gives PFC a secret rank if you play way too much tgmc

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
the ultimate reward that darius and David asked for if you play way too much tgmc (1000 hours)

## Why It's Good For The Game
the ultimate reward to ~~laugh at people who play way too much~~ REAL determined gamers.

## Changelog
:cl:
add: Added cool rank to squad marine if you play a ton. 
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -53. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Hughgent** [comment]
Not the E4 mafia?

**Comment by Pariah919** [comment]
@Hughgent e4 has a lot of overlap already ~~and isn't as funny as this~~

File: code/datums/jobs/job/marines.dm [hunk 1]
```diff
@@ -50,8 +50,10 @@ Make your way to the cafeteria for some post-cryosleep chow, and then get equipp
 			new_human.wear_id.paygrade = "E2"
 		if(6601 to 18000)
 			new_human.wear_id.paygrade = "E3"
-		if(18001 to INFINITY)
+		if(18001 to 60000)
 			new_human.wear_id.paygrade = "E3E"
+		if(60001 to INFINITY)
+			new_human.wear_id.paygrade = "E8" //If you play way too much TGMC. 1000 hours.
```
Comment (spookydonut) [inline code/datums/jobs/job/marines.dm:56]:
```suggestion
		if(18001 to 600000)
			new_human.wear_id.paygrade = "E3E"
		if(600001 to INFINITY)
			new_human.wear_id.paygrade = "E8" //If you play way too much TGMC. 10000 hours.
```

**Review by spookydonut — CHANGES_REQUESTED** [review]
or no dice.

**Comment by spookydonut** [comment]
lmao i was shitposting.

**Comment by Hughgent** [comment]
1 billion hours or no merge

**Review by Hughgent — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-54). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**